pkgs/development/libraries/freetype/default.nix

```diff
@@ -65,9 +75,10 @@ stdenv.mkDerivation (finalAttrs: {
   postInstall = glib.flattenInclude
     # pkgsCross.mingwW64.pkg-config doesn't build
     # makeWrapper doesn't cross-compile to windows #120726
-    + lib.optionalString (!stdenv.hostPlatform.isMinGW) ''
+    + ''
     substituteInPlace $dev/bin/freetype-config \
       --replace ${buildPackages.pkg-config} ${pkgsHostHost.pkg-config}
+  '' + lib.optionalString (!stdenv.hostPlatform.isWindows) ''
```

Comment:
I'm seeing `isMinGW -> isWindows`. Does the fix also apply for Cygwin?

Reply:
That was a mistake. Only meant to do that for mingw. Force pushed to fix.